rs/execution_environment/src/scheduler.rs

```diff
@@ -909,6 +913,7 @@ impl SchedulerImpl {
         subnet_size: usize,
     ) -> (
         Vec<CanisterState>,
+        BTreeSet<CanisterId>,
```

Comment:
Doc comment above should be updated too.

Reply:
Done.